modules/ROOT/pages/getting-started.adoc

```diff
@@ -2,16 +2,17 @@
 
 == Introduction
 
-=== Update Streams
+=== Streams
 
-There are three Fedora CoreOS (FCOS) update streams available: `stable`, `testing`, and `next`. In general, you will want to use `stable`, but it is recommended to run some machines on `testing` and `next` as well.
+Metadata about Fedora CoreOS is available in a custom JSON format, called "stream metadata".  For maintaining automation, it is expected that you will interact with this stream metadata.  There are three Fedora CoreOS (FCOS) update streams available: `stable`, `testing`, and `next`. In general, you will want to use `stable`, but it is recommended to run some machines on `testing` and `next` as well and provide feedback.
 
 Each stream has a canonical URL representing its current state in JSON format.  For example, the URL for the stable stream is: https://builds.coreos.fedoraproject.org/streams/stable.json
 
 While Fedora CoreOS does automatic in-place updates, it is generally a good practice to start provisioning new machines from the latest images.
 The format of this JSON file is stable, and you can parse it to e.g. find the latest ISO/AMI/etc.
```

Comment:
Not from your PR, though we might as well drop this sentence too now. It looked out of order even before, but now we have that info in the stream metadata page anyway.